angularjs-portal-frame/src/main/webapp/portal/main/partials/welcome-modal-template.html

```diff
@@ -0,0 +1,12 @@
+<div class="welcome-popup center">
+  <h1 ng-bind-html="welcome.title"></h1>
+  <img src="{{ welcome.img }}" class="hidden-xs">
+	<div class="sr-only" tabindex="0" id="welcomeContent" aria-labelledby="welcomeContent">
```

Comment:
if you want the screen reader just read the title or let them know it is a welcome pop-up window, you might want to add the id on the title(not the full text area) and use aria-labelledby; or if you want it reads all the text like a paragraph as long as the window pop-up, use a aria-describedby. My suggestion might be first let them know there is an welcome pop-up window, then they might use TAB to focus on the text. This might be helpful:http://www.w3.org/TR/wai-aria/states_and_properties